product_docs/docs/transporter/ba_preview/limitations.mdx

```diff
@@ -19,6 +19,6 @@ Unsupported Oracle data types include:
 - XML
 - Spatial
 
-Don't use EDB Transporter for replicating Oracle tables that contain blob, clob, or nclob columns and are expected to get UPDATE changes. Streaming UPDATE changes currently aren't handled correctly.
+EDB Data Migration Service supports replicating Oracle tables that contain BLOB, CLOB, or NCLOB columns only if these also have the `PRIMARY KEY` constraint. If the tables don;t have the `PRIMARY KEY` constraint, the streaming replication will only support INSERT operations.
```

Comment:
@yidian1997  I saw you had a conversation with @ksnavely about this in the other thread (https://github.com/EnterpriseDB/docs/pull/5908/files#r1703232801). Is this wording more accurate?

Reply:
```suggestion
EDB Data Migration Service supports replicating Oracle tables that contain BLOB, CLOB, or NCLOB columns only if these also have the `PRIMARY KEY` constraint. If the tables don't have the `PRIMARY KEY` constraint, the streaming replication will only support INSERT operations.
```